scripts/calc.py

```diff
@@ -0,0 +1,23 @@
+"""
+usage: teuthology-calc [-h] [options] [<suite_path>...]
+
+Print suite config description, matrix dump, file list
```

Comment:
What is the purpose of this cli? what is it useful for?  Feels like part of it is same as teuthology-describe?

Reply:
@vasukulkarni the teuthology-describe does not do the same.

Reply:
@vasukulkarni from the descriptions of describe-tests:
```
Describe the contents of a qa suite by reading 'meta' elements from
yaml files in the suite.

The 'meta' element should contain a list with a dictionary
of key/value pairs for entries, i.e.:
```
The calc has nothing to do with 'meta' elements, it just shows the teuthology forensic data, exactly what is parsed by teuthology.suite.build_matrix.